cmd/expandybird/expander/expander_test.go

```diff
@@ -16,168 +16,664 @@ limitations under the License.
 
 package expander
 
-/*
 import (
 	"fmt"
-	"io"
-	"io/ioutil"
-	"os"
-	"path"
 	"reflect"
+	"runtime"
 	"strings"
 	"testing"
 
+	"github.com/kubernetes/helm/pkg/chart"
 	"github.com/kubernetes/helm/pkg/common"
-	"github.com/kubernetes/helm/pkg/util"
 )
 
-var importFileNames = []string{
-	"../test/replicatedservice.py",
-}
-
-var validFileName = "../test/ValidContent.yaml"
-var outputFileName = "../test/ExpectedOutput.yaml"
-var archiveFileName = "../test/TestArchive.tar"
 var expanderName = "../../../expansion/expansion.py"
 
-type ExpanderTestCase struct {
+type testCase struct {
 	Description      string
-	TemplateFileName string
-	ImportFileNames  []string
+	Request          *common.ExpansionRequest
+	ExpectedResponse *common.ExpansionResponse
 	ExpectedError    string
 }
 
-func (etc *ExpanderTestCase) GetTemplate(t *testing.T) *common.Template {
-	template, err := util.NewTemplateFromFileNames(etc.TemplateFileName, etc.ImportFileNames)
-	if err != nil {
-		t.Fatalf("cannot create template for test case '%s': %s", etc.Description, err)
-	}
+// content provides an easy way to provide file content verbatim in tests.
+func content(lines []string) []byte {
+	return []byte(strings.Join(lines, "\n") + "\n")
+}
```

Comment:
You can also do this with literals, which is common and may be more readable, like this:

```go
var data []byte(`
    foo:
        foo1:bar1
        foo2:bar2
    bar: baz
`)

```